charts/shoot-core/components/charts/network-policies/templates/allow-to-seed.yaml

```diff
@@ -0,0 +1,21 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Egress from pods labeled with 'networking.gardener.cloud/to-seed=allowed'
+      to the seed.
+  name: gardener.cloud:system:allow-to-seed
+  namespace: kube-system
+  labels:
+    origin: gardener
+spec:
+  podSelector:
+    matchLabels:
+      networking.gardener.cloud/to-seed: allowed
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - ipBlock:
+        cidr: 0.0.0.0/0
```

Comment:
this is not `to seed` but `to everything`. 
Could not you use the in-cluster IP of the API?
It even can be calculated by the botanist, see an example:
https://github.com/gardener/gardener/blob/8a23fe8d42a3edbb9ca9c79f2beec95275677dfb/pkg/operation/botanist/monitoring.go#L95

Reply:
> this is not to seed but to everything.

Maybe rename it to `allow-to-public` (similar to the network policies in the seed)?

> Could not you use the in-cluster IP of the API?

I don't think that's possible. For network policies the "real" IP must be used. The service network is just resulting in iptable rules.

Reply:
> Could not you use the in-cluster IP of the API?
No, I tried creating a network policy with the `kubernetes` service IP and it gets blocked because it needs the real IP as @rfranzke mentioned. I changed the name to `allow-to-public-networks` to stay consistent to the network policies we have in the seeds.

Reply:
Ok, I agree. Thanks!